### PR TITLE
Vue router

### PR DESCRIPTION
**Pull request by rlajous**
## Summary
- Added login page
- Router-link to redirect between screens
- Request in login page

## Screenshot
login

## Trello Card
https://trello.com/c/luLpiW9a/18-vue-router

### CODE REVIEW

File: wolox-books/src/config/api.js [hunk 1]
```diff
@@ -1,8 +1,8 @@
-import { create } from "apisauce";
+import { create } from 'apisauce'
 
 const api = create({
-  baseURL: "https://wbooks-api-stage.herokuapp.com/api/v1",
+  baseURL: 'https://wbooks-api-stage.herokuapp.com/api/v1',
```
Comment (FrankIglesias) [inline wolox-books/src/config/api.js]: this can be moved to a .env file

File: wolox-books/src/views/Login.vue [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+    .error(v-if='!$v.email.required && submitStatus')
+      |Email is required
+    .error(v-if='!$v.email.email')
+      |Pleas enter a valid email
+    label.label(for='password')
+      |{{$t('register.password')}}
+    input#password(type='password' name='password' v-model='password')
+    .error(v-if='!$v.password.required && submitStatus')
+      |Password is required
+    button
```
Comment (FrankIglesias) [inline wolox-books/src/views/Login.vue:16]: where is the class of this button?

File: wolox-books/src/views/Login.vue [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+    email: {
+      required,
+      email
+    }
+  }
+}
+</script>
+
+<style lang='scss'>
+
```
Comment (FrankIglesias) [inline wolox-books/src/views/Login.vue]: remove this line

File: wolox-books/src/views/Login.vue [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+  data () {
+    return {
+      email: null,
+      password: null,
+      submitStatus: false
+    }
+  },
+  methods: {
+    onSubmit () {
+      authService.logIn({
```
Comment (FrankIglesias) [inline wolox-books/src/views/Login.vue]: you are not checking the form validity before the service calling

**Review by FrankIglesias — CHANGES_REQUESTED** [review]

File: wolox-books/src/views/Login.vue (hunk 2 above)
Comment (rlajous) [reply wolox-books/src/views/Login.vue:16]: I will style everything after

File: wolox-books/src/views/Login.vue [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+  },
+  methods: {
+    onSubmit () {
+      this.$v.$touch()
+      this.submitStatus = this.$v.$invalid
+      if (!this.submitStatus) {
+        logIn({
+          session: {
+            email: this.email,
+            password: this.password
```
Comment (mcallegari10) [inline wolox-books/src/views/Login.vue]: Use  destructuring

**Review by mcallegari10 — CHANGES_REQUESTED** [review]

**Review by mcallegari10 — APPROVED** [review]

**Pull request merged**